ocrd_monitor/Dockerfile

```diff
@@ -19,6 +19,13 @@ RUN apt-get update \
 ENV GDK_BACKEND broadway
 ENV BROADWAY_DISPLAY :5
 
+# environment variables used by flask application
+ENV APP_BASEDIR=/data
+# broadway 
+ENV APP_BW_PORT=8085
+# logviewer
+ENV APP_LOG_PORT=8088
+
 EXPOSE 8085
 EXPOSE 8080
 EXPOSE 5000
```

Comment:
If we define these as vars, we should also use them – here in the `EXPOSE` and below in `CMD`.

Reply:
Expose and Env happen to be the same ports. Env ports can be configured differently by the host but the Expose are fixed.